pulsar-io/kafka-connect-adaptor/src/main/java/org/apache/pulsar/io/kafka/connect/PulsarOffsetBackingStore.java

```diff
@@ -136,9 +141,6 @@ void processMessage(Message<byte[]> message) {
     @Override
     public void start() {
         try {
-            client = PulsarClient.builder()
-                .serviceUrl(serviceUrl)
-                .build();
             log.info("Successfully created pulsar client to {}", serviceUrl);
```

Comment:
this log line is irrelevant now

Reply:
please remove `serviceUrl` - it is no longer needed and adjust the rest of logging accordingly.
Can remove it from the config definitions as well.

Reply:
Yeah, plan to remove the `serviceUrl` after the current unit test fix. Thanks for the reminding.